apps/web/src/instrumentation.ts

```diff
@@ -0,0 +1,9 @@
+import { registerInstance } from '@documenso/lib/server-only/telemetry/register-instance';
+
+import packageInfo from '../package.json';
+
+export const register = async () => {
+  if (process.env.NEXT_RUNTIME === 'nodejs') {
+    await registerInstance({ version: packageInfo.version });
+  }
+};
```

Comment:
At the moment, there's no issue in the development environment - it's functioning well. However, when I build the project and run it using npm run start, I encounter an Internal Server Error. I've been investigating this problem since yesterday but haven't been able to resolve it. I could use some assistance in figuring out what's causing this error.